src/ucp/core/ucp_ep.h

```diff
@@ -477,6 +477,9 @@ struct ucp_ep_config {
     /* Bitmap of lanes selected by the protocols */
     ucp_lane_map_t                proto_lane_map;
 
+    /* Indicate that protocol initialization was done for config */
+    int                           is_proto_initialized;
```

Comment:
maybe define common flags field + enum with one element (so we can use/add more flags in the future)